src/test/java/uk/gov/dhsc/htbhf/steps/ClaimScenario.java

```diff
@@ -12,6 +12,7 @@
     IDENTITY_NOT_MATCHED("claim-identity-failed.json"),
     ELIGIBILITY_NOT_CONFIRMED("claim-eligibility-failed.json"),
     DUPLICATE_CLAIM("claim-duplicate.json"),
+    CHILD_DOB_MISMATCH("child-dob-mismatch.json"),
```

Comment:
I've been using the prefix `claim` for any responses from the claim endpoint so its easier to tell them apart from the postcode or other mocked response, so can we call this `claim-child-dob-mismatch.json`?